src/filter/levels/levels.c

```diff
@@ -136,38 +159,50 @@ void f0r_destruct(f0r_instance_t instance)
   free(instance);
 }
 
-void f0r_set_param_value(f0r_instance_t instance, 
+static int ClampIntToRange(double input, int min_allowed, int max_allowed)
```

Comment:
There is a CLAMP macro in include/frei0r_math.h.

Reply:
Cool. Using that now.